apps/communications/dialer/js/navigation.js

```diff
@@ -0,0 +1,45 @@
+(function(exports) {
+  'use strict';
+
+  var currentView = null;
+
+  exports.Navigation =  {
+    get currentView() {
+      return currentView ? currentView.id : null;
+    },
+    go: function(destination) {
+      var view =
+        document.querySelector(
+          'div[aria-labelledby="' + destination + '"]'
+        );
```

Comment:
nit: I'm not super-fond of querying via the aria-labelledby attribute, besides if the view is not found we return silently without throwing an error or anything. I'd rather have a list of the possible destinations hardcoded into the Navigation object with clear short names (keypad, recents, contacts) which would then be mapped to the id. If ``destination`` is not among the list then we throw an error. E.g., from the outside:
```javascript
Navigation.show('keypad');
```
And internally:
```javascript
go:function(destination) {
  var view;
  switch (destination) {
    case 'keypad': view = document.getElementyById('keypad-panel'); break;
    // ...
    default: throw new Error("Navigating to an invalid destination: " + destination);
  }
}